system/CLI/CLI.php

```diff
@@ -516,6 +516,37 @@ public static function color(string $text, string $foreground, string $backgroun
 			$string .= "\033[4m";
 		}
 
+		// Detect if color method was already in use with this text
+		if (strpos($text, "\033[0m") !== false)
+		{
+			// Split the text into parts so that we can see
+			// if any part missing the color definition
+			$chunks = mb_split("\\033\[0m", $text);
+			// Reset text
+			$text = '';
+
+			foreach ($chunks as $chunk)
+			{
+				if (empty($chunk))
```

Comment:
`0` can be marked as empty, probably use empty string `''` check

Reply:
Good point, thanks.